app/views/cash_account_declaration_details_search_no_result.scala.html

```diff
@@ -0,0 +1,91 @@
+@*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this(
+       main_template: Layout,
+       h2: components.h2,
+       h1Inner: components.h1Inner,
+       h2Inner: components.h2Inner,
+       p: components.p,
+       ul: components.unorderedList,
+       link: components.link,
+       newTabLink: components.newTabLink
+)
+
+@(
+    pageNumber: Option[Int],
+    accountNumber: String,
+    searchInput: String
+)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
```

Comment:
if we're doing our params line by line it might be worth doing it for the implicits as well

Reply:
Done